gaeguli/pipeline.c

```diff
@@ -67,6 +67,10 @@ struct _GaeguliPipeline
 
   gboolean prefer_hw_decoding;
 
+  GstDeviceMonitor *device_monitor;
+  GMainLoop *loop;
+  guint target_node_id;
```

Comment:
Why **target**_node_id? This is the node ID allocated for `GaeguliPipeline`, not `GaeguliTarget`, so the name feels somewhat misleading. Change it to `pw_node_id` maybe?